packages/headless/src/features/facet-options/facet-options-state.ts

```diff
@@ -1,9 +1,17 @@
-import {FacetOptions} from './facet-options';
```

Comment:
I'm no longer importing `FacetOptions`, since it corresponds with the search request and I dissociated `facet-options-*` from the search request.